.eslintrc.js

```diff
@@ -7,10 +7,6 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:@typescript-eslint/recommended',
-    'plugin:vue/base',
```

Comment:
Removed Vue2 specific plugins

Reply:
Are these for Vue2 only, or did they bring value for Vue3 code?

Reply:
Are there updated plugins for Vuetify 3?

Reply:
I read the documentation again `vue/base` should be included. No updates for Vuetify 3

Reply:
Perfect, thanks!